ardriveapp/turbo-sdk
# chore(PE-5886): fix docker-compose for integration tests against local s…

…erivces

Fixing these so we can pull in `arlocal` and test interactions related to AR credit payments

Steps to validate it's working
- Create a `.env` file and add the STRIPE_SECRET_KEY from the `test` environment (`sk_test_XXX`)
- `yarn test:docker`

It should spin up the upload and payment service locally, and pass all tests.
